README.md

```diff
@@ -120,14 +120,26 @@ Explicit steps for setting up CCM in a development environment.
     2. In the "Configuration Type" menu, select "By Client ID".
     3. In the "Client ID" field, paste in the same ID number that was added to `.env` above.
     4. When prompted to verify the ID for the tool, click the "Install" button.
+#### Database Migrations
+
+Running migrations using the Umzug. Umzug is a sister library to Sequelize for migration tasks. More info [refer](https://github.com/sequelize/umzug#cli-usage). After creating/modifying model you create a migration file and run migration. Create a model using Sequelize ORM and create/run migration using Umzug CLI. These steps needs to be done by the dev's.
+
+1. Running migration `Locally`
+       1. Running the migrations `docker exec -it ccm_web node -r ts-node/register server/src/migrator up`
+       2. Reverting the migration `docker exec -it ccm_web node -r ts-node/register server/src/migrator down`.
+       3. Create a migration file `docker exec -it ccm_web node -r ts-node/register server/src/migrator create --name my-migration.ts`. Generate a migration file called `<timestamp>.my-migration.ts` The timestamp prefix can be customized to be date-only or omitted, but be aware that it's strongly recommended to ensure your migrations are lexicographically sortable so it's easy for humans and tools to determine what order they should run in - so the default prefix is recommended.
+2. Running the migration are usually done when server is starting up, but in addition if you want to run migrations or revert use above commands
+3. Running migrations `dev/test/prod`
```

Comment:
`dev/test/prod` is confusing to me. I think we can just say when using the Docker Compose production workflow or `docker-compose-prod.yml`.

Reply:
Done